packages/runtime/src/context.ts

```diff
@@ -136,6 +140,18 @@ export function createContext(params: ContextParams): Context {
       await doc!.replace(content, typeof shouldIndex === 'undefined' ? undefined : { shouldIndex })
       return doc
     },
+    enableDocIndexing: async <Content extends Record<string, any> = Record<string, any>>(
+      model: string,
+      opts?: UpdateOpts,
+    ): Promise<void> => {
+      const controller = getViewerID()
+      if (controller == null) {
+        throw new Error('Document can only be hidden with an authenticated account')
+      }
+
+      const doc = await loader.load<Content>({ id: model })
+      await doc.shouldIndex(false, opts)
```

Comment:
The `shouldIndex` argument should be provided here so indexing can be disabled but also enabled back
```suggestion
      id: string,
      shouldIndex: boolean,
      opts?: UpdateOpts,
    ): Promise<void> => {
      const controller = getViewerID()
      if (controller == null) {
        throw new Error('Document can only be hidden with an authenticated account')
      }

      const doc = await loader.load<Content>({ id })
      await doc.shouldIndex(shouldIndex, opts)
```